models/User.js

```diff
@@ -5,17 +5,17 @@ const cleanUser = (user) => {
     return user ?
         {
             ...user,
-            passwordHash: 'hidden'
+            password: 'hidden'
         } :
         undefined;
 };
  
  
-User.findByEmailAndPassword = (email, password, callback) => {
+User.findByUsername = (username, password, callback) => {
```

Comment:
password is not used anymore: remove it ;)

Reply:
Login is not working anymore when i remove it.